_drafts/programatically-configured-bots.md

```diff
@@ -52,9 +52,9 @@ robot.alias = '/'
 # like robot name
 robot.adapter.once 'connected', ->
 
-  # require external hubot scripts
-  robot.loadScriptPackage require('hubot-help')
-  robot.loadScriptPackage require('hubot-shipit')
+  # require external hubot script packages
+  robot.loadPackage 'hubot-help'
+  robot.loadPackage 'hubot-shipit'
```

Comment:
I would strongly recommend to not pass strings, but the actual function. Once there is an instance I would try to avoid having dynamic requires as much as possible. So I would do 

```js
robot.loadPackage(require('hubot-help'))
```

instead of 

```js
robot.loadPackage('hubot-help')
```

For example, with the `hubot` package is linked into my app, the above command will fail, because it cannot find the `hubot-help` package inside the `hubot` directory, unless I manually set `NODE_PATH` to include by app’s `node_modules` path. 

In case we decide to do it that way, it might make sense to rename `robot.loadPackage` to just `robot.load`?